modules/imaging_browser/php/imagingbrowserrowprovisioner.class.inc

```diff
@@ -166,7 +171,7 @@ class ImagingBrowserRowProvisioner extends \LORIS\Data\Provisioners\DBRowProvisi
         // $scan_types are set in the configuration module
         $modalities_subquery = '';
         foreach ($case_desc as $key => $value) {
-            $modalities_subquery = "$value as $scan_id_types[$key]_QC_Status";
+            $modalities_subquery = "$value as `$scan_id_types[$key]_QC_Status`";
```

Comment:
```suggestion
            $modalities_subquery = $value." as ".$instance->quote("$scan_id_types[$key]_QC_Status");
```

Reply:
You're right, on L174 this will work, it's my fault for missing it, however trying to use quote() on https://github.com/aces/Loris/blob/23.0-release/modules/imaging_browser/php/imagingbrowserrowprovisioner.class.inc#L155

Like this:

```php
GROUP BY files.SessionID) ". $instance->quote("$pass[$key] ") .
```
or 
```php
GROUP BY files.SessionID) ". $instance->quote($pass[$key] ) .
```
won't work, as it returns the following error:
>Syntax error or access violation: 1064 You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near '**'t1pass '** ON (\`t1pass\`.SessionID=f.SessionID\n            ) \n            LEFT JOI' at line 76 

and on [L156](https://github.com/aces/Loris/blob/23.0-release/modules/imaging_browser/php/imagingbrowserrowprovisioner.class.inc#L156), we need to use backticks on ``" ON ( `$pass[$key]`.SessionID=f.SessionID`` otherwise `$pass[$key]` will be interpreted as a string literal instead of identifier, and the comparison will not work.